### PR TITLE
typo fixes.

### PR DESCRIPTION
**Pull request by ePaul**
I did run aspell over this (ignoring a lot of words ... it looks like there isn't a markdown filter yet for it).

I can also split this up to several commits (or PRs), if wanted.

### CODE REVIEW

**Comment by webron** [comment]
Thanks, @ePaul. Some are not needed. I'll comment on those and merge after the changes are done.

File: versions/3.0.md [hunk 1]
```diff
@@ -575,7 +575,7 @@ Describes a single API operation on a path.
 Field Name | Type | Description
 ---|:---:|---
 <a name="operationTags"></a>tags | [`string`] | A list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier.
-<a name="operationSummary"></a>summary | `string` | A short summary of what the operation does. For maximum readability in the swagger-ui, this field SHOULD be less than 120 characters.
+<a name="operationSummary"></a>summary | `string` | A short summary of what the operation does. For maximum readability in the swagger-UI, this field SHOULD be less than 120 characters.
```
Comment (webron) [inline versions/3.0.md]: not needed

File: versions/3.0.md [hunk 2]
```diff
@@ -849,7 +849,7 @@ A path parameter of a string value:
 {
   "name": "username",
   "in": "path",
-  "description": "username to fetch",
+  "description": "user name to fetch",
```
Comment (webron) [inline versions/3.0.md]: not needed

File: versions/3.0.md [hunk 3]
```diff
@@ -860,7 +860,7 @@ A path parameter of a string value:
 ```yaml
 name: username
 in: path
-description: username to fetch
+description: user name to fetch
```
Comment (webron) [inline versions/3.0.md]: not needed

File: versions/3.0.md [hunk 4]
```diff
@@ -1585,7 +1585,7 @@ myWebhook:
               $ref: '#/components/definitions/SomePayload'
       responses:
         200:
-          description: webhook successfully processed an no retries will be performed
+          description: web hook successfully processed an no retries will be performed
```
Comment (webron) [inline versions/3.0.md]: not needed

File: versions/3.0.md [hunk 5]
```diff
@@ -1641,7 +1641,7 @@ X-Rate-Limit-Reset:
 
 #### <a name="exampleObject"></a>Example Object
 
-Allows sharing examples for operation requests and responses. This object can either be a freeform object, array or primitive value.  To represent examples of media types that cannot naturally represented in the OpenAPI definition, a string value can be used to contain the example with escaping where necessary. 
+Allows sharing examples for operation requests and responses. This object can either be a free form object, array or primitive value.  To represent examples of media types that cannot naturally represented in the OpenAPI definition, a string value can be used to contain the example with escaping where necessary.
```
Comment (webron) [inline versions/3.0.md]: not needed

File: versions/3.0.md [hunk 6]
```diff
@@ -1793,7 +1793,7 @@ paths:
     - name: id
       in: path
       required: true
-      description: the user identifier, as userId or username
+      description: the user identifier, as user Id or user name
```
Comment (webron) [inline versions/3.0.md]: not needed

File: versions/3.0.md (hunk 2 above)
Comment (ePaul) [inline versions/3.0.md]: I'm no native speaker of English, but I thought English would use much less composite words than my mother tongue (German).
Is this an actual English word here, or just repeating the variable name?

**Comment by ePaul** [comment]
Removed the changes mentioned as "not needed".

File: versions/3.0.md (hunk 2 above)
Comment (webron) [inline versions/3.0.md]: I'm no native speaker as well, but when it comes to these cases, I use a google search for the word (if needed, add a "define" before the word) and see if it comes up with a result. `username` is commonly used today, and there's no reason to break it down.

**Comment by RobDolinMS** [comment]
Thanks for doing this @ePaul.  

As a dog owner, my favorite typo is "mutlipart" ;)

**Pull request merged**